CMakeLists.txt

```diff
@@ -60,66 +58,36 @@ MESSAGE(${PYTHON_INCLUDE_DIRS})
 #  Boost library
 #
 
-MESSAGE("Looking for Boost libraries...")
+MESSAGE(STATUS "Looking for Boost libraries...")
 SET(Boost_NO_BOOST_CMAKE TRUE) # to fix the problem of finding python37 libs
 SET(Boost_USE_STATIC_LIBS=OFF)
 SET(Boost_USE_MULTITHREADED=OFF)
 SET(Boost_USE_STATIC_RUNTIME=OFF)
 
-# Adapt it according to your system
-#FIND_PACKAGE(Boost 1.73.0 REQUIRED)
-FIND_PACKAGE(Boost 1.82.0 REQUIRED)
-#FIND_PACKAGE(Boost REQUIRED)
+FIND_PACKAGE(Boost 1.74.0 REQUIRED COMPONENTS regex python${LIBRA_Python3_VERSION})
 IF(Boost_FOUND)
-  MESSAGE("Success!")
-  INCLUDE_DIRECTORIES("${Boost_INCLUDE_DIRS}") 
-  # Adapt it based on your system
-  #FIND_PACKAGE(Boost COMPONENTS python36 regex) # it seems that it is important to find sub-packages too!!!
-  FIND_PACKAGE(Boost COMPONENTS python39 regex)

```

Comment:
it is important to be able to find those Boost components so this can not be deleted

Reply:
See line with find_package statement: it will look for theses components.